examples/rust/poseidon2/Readme.md

```diff
@@ -0,0 +1,5 @@
+# Using Poseidon2 API
+
+Sanity checks with standard sage output from attached sage script. The sage script originally from [Horizen labs code](https://github.com/HorizenLabs/poseidon2/blob/main/poseidon2_rust_params.sage) has been modified to print in standard format.
```

Comment:
We should use a permalink to avoid linking to the latest script and always link to the script we used

Reply:
done